up & going/ch2.md

```diff
@@ -218,7 +218,7 @@ b;              // 42 -- o número!
 
 #### Truthy & Falsy
 
-No Capítulo 1, nós mencionamos a natureza "truthy" e "falsy" dos valores: quando um valor não-`boolean` é coergido para um valor `boolean`, eke se tornam `true` ou `false`, respectivamente?
+No Capítulo 1, nós mencionamos a natureza "truthy" e "falsy" dos valores: quando um valor não-`boolean` é coergido para um valor `boolean`, eles se tornam `true` ou `false` respectivamente?
```

Comment:
No Capítulo 1, nós mencionamos brevemente a natureza "truthy" e "falsy" dos valores: quando um valor não-`boolean` é ~~coergido~~ coagido para um valor `booleano`, ele~~s~~ se torna~~m~~ **de fato** `true` ou `false`~~respectivamente~~?

(In Chapter 1, we briefly mentioned the "truthy" and "falsy" nature of values: when a non-boolean value is coerced to a boolean, does it become true or false, respectively?)

"Ele" se refere ao valor, que está no singular ("quando _um valor_ ..., _ele_ se torna ...?").